lib/Api/CampaignApi.php

```diff
@@ -17,7 +17,7 @@
  * OpenAPI spec version: v1
  * 
  * Generated by: https://github.com/swagger-api/swagger-codegen.git
- * Swagger Codegen version: 2.4.0-SNAPSHOT
+ * Swagger Codegen version: 2.4.10
  */
```

Comment:
This file is huge, it's mostly just the array/stdclass check, but there was some other parameters added to some calls later in the file (I added a separate comment where that is).